src/DeviceInterfaceValues.tsx

```diff
@@ -531,8 +532,10 @@ const UnsetDataModal = ({
   unsettingData,
   handleShowUnsetModal,
   unsetInterfaceData,
+  propertyData,
 }: UnsetDataModalProps) => {
   const [selectedEndpoint, setSelectedEndpoint] = useState<string>('');
+  const [endpoints, setEndpoints] = useState<string[]>([]);
```

Comment:
`endpoints` is too generic here. Which kind of endpoint are those? All the endpoints? only the ones with some data? only the ones that can be unset?